content/soarian/dstu2/general-clinical/procedure.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Procedure | Soarian DSTU 2 API
+---
+ 
+# Procedure
+
+* TOC
+{:toc}
+
+## Overview
+
+The API returns procedures charted in Soarian Clinicals. Procedures in completed or aborted status are returned; procedures in erroneous status are not returned. Note that traditional charting practices have procedures resolved and charted in coding and billing systems and not interfaced back to the EMR; therefore, the procedure record in Soarian may not be complete nor precise to a procedure list provided as part of a patient's billing record.  
+
+## Terminology Bindings
+
+<%= terminology_table(:soarian_procedure, :dstu2) %>
+
+## Custom Extensions
+
+The [bundle extension] can be returned, see possible codes in the [errors section] below.
+
+## Search
+
+    GET /Procedure?:parameters
+    
+_Implementation Notes_
+
+* The API returns the most current procedure list.
+
+* Procedures in completed or aborted status are returned.
+
+* Procedures in erroneous status are not returned.
```

Comment:
Yes. In Soarian, there is no concept of in-progress.
In Procedure valid status are Completed,Aborted and Erroneous.
API returns only Completed or Aborted.